index.js

```diff
@@ -100,8 +100,10 @@ global.window = {
   customEvent: () => {}
 }
 
-const Tonic = require('@socketsupply/tonic')
+const { Tonic } = await import('@socketsupply/tonic')
```

Comment:
is it possible to use static import on top of the module?

Reply:
Unfortunately it appears that the approach here is loading up the global object with some expected values before importing so I think it would throw with a static import. I'll double check though.

Reply:
Yup, static import doesn't work. I have some ideas that might make tonic just work everywhere.